cpp/examples/deck.gl/flight-paths.cc

```diff
@@ -58,7 +58,7 @@ auto createLineLayer(const std::string &dataPath) -> std::shared_ptr<LineLayer::
   };
   lineLayerProps->getColor = [](const Row &row) -> mathgl::Vector4<float> {
     // NOTE: Decimal array data is being loaded as double
-    float z = static_cast<float>(row.getDoubleVector3("start").y);
+    float z = static_cast<float>(row.getDoubleVector3("start").z);
     float r = z / 10000.0f;
     return {255.0f * (1.0f - r * 2.0f), 128.0f * r, 255.0f * r, 255.0f * (1.0f - r)};
```

Comment:
In JS we assign this to Uint8ClampedArrays so we get automatic clamping. As you say we need another way to handle it here.